core/src/main/java/nl/nn/adapterframework/configuration/digester/ValidateAttributeRule.java

```diff
@@ -27,37 +27,48 @@
 
 import nl.nn.adapterframework.configuration.ConfigurationWarning;
 import nl.nn.adapterframework.configuration.HasSpecialDefaultValues;
-import nl.nn.adapterframework.configuration.SuppressKeys;
 import nl.nn.adapterframework.doc.Protected;
-import nl.nn.adapterframework.util.AppConstants;
 import nl.nn.adapterframework.util.EnumUtils;
 import nl.nn.adapterframework.util.StringResolver;
 
 /**
  * @author Niels Meijer
  */
 public class ValidateAttributeRule extends DigesterRuleBase {
-	private boolean suppressDeprecationWarnings = AppConstants.getInstance().getBoolean(SuppressKeys.DEPRECATION_SUPPRESS_KEY.getKey(), false);
 
+	/**
+	 * @see DigesterRuleBase#handleBean()
+	 */
 	@Override
 	protected void handleBean() {
-		if(!suppressDeprecationWarnings) {
 			Class<?> clazz = getBeanClass();
 			ConfigurationWarning warning = AnnotationUtils.findAnnotation(clazz, ConfigurationWarning.class);
 			if(warning != null) {
 				String msg = "";
-				if(AnnotationUtils.findAnnotation(clazz, Deprecated.class) != null) {
+				boolean isDeprecated = AnnotationUtils.findAnnotation(clazz, Deprecated.class) != null;
+				if(isDeprecated) {
```

Comment:
Klopt het dat er alleen maar een warning komt als er naast de Deprecated annotatie ook nog een ConfigurationWarning annotatie is? Er moet ook een warning komen als er alleen maar een Deprecated annotatie is.

Reply:
> Klopt het dat er alleen maar een warning komt als er naast de Deprecated annotatie ook nog een ConfigurationWarning annotatie is? Er moet ook een warning komen als er alleen maar een Deprecated annotatie is.

Ik heb dit met Niels besproken; omdat er bij de `Deprecated` warning geen tekst meegegeven kan worden als hint voor de gebruiker moet een `Deprecated` warning altijd vergezeld gaan van een `ConfigurationWarning` als deze in de configuration-warnings aan de gebruiker getoond moet worden.